### PR TITLE
PICARD 776: Delete additional files when moving

### PR DESCRIPTION
**Pull request by cheath2013**
U.I. option identical to "move additional files" created that has the ability to delete additional files, based on the pattern provided. 

Deletes files indefinitely, does not send to recycling bin. As a safeguard for accidental deletions I can create a dialog box using Tkinter that asks the user to confirm delete of type 'type' from path 'path' (Not included in this pull request, but available on request).

### CODE REVIEW

File: picard/file.py [hunk 1]
```diff
@@ -360,6 +364,27 @@ def _move_additional_files(self, old_filename, new_filename):
...
+            # FIXME glob1 is not documented, maybe we need our own implementation?
+            for old_file in glob.glob1(old_path, pattern):
+                old_file = os.path.join(old_path, old_file)
+                # FIXME we shouldn't do this from a thread!
+                if self.tagger.files.get(decode_filename(old_file)):
+                    """Ensures file being saved is not deleted"""
+                    log.debug("File loaded in the tagger, not deleting %r", old_file)
+                    continue
+                log.debug("Deleting %r", old_file)
+                os.remove(old_file)#removes file from existance.
```
Comment (zas) [inline picard/file.py]: Add two spaces before comment `#` or even remove the whole comment, it doesn't help much.

**Comment by zas** [comment]
> Deletes files indefinitely, does not send to recycling bin. As a safeguard for accidental deletions I can create a dialog box using Tkinter that asks the user to confirm delete of type 'type' from path 'path' (Not included in this pull request, but available on request).

Imho, a confirmation is very needed as such feature may lead to data loss.

A confirmation dialog with list of files to be deleted, with Delete/Cancel buttons.
Another boolean option to enable/disable the confirmation dialog (enabled by default of course) should also be added to Settings.

Btw, i don't see what TkInter does here, i guess you meant PyQt ?

**Comment by cheath2013** [comment]
@zas I attempted using PyQt to create a dialog box but got the following error: "Qpixmap: It is not safe to use pixmaps outside the gui thread". Because I struggled to find a way around this error, I looked elsewhere and found that I could do what I needed to with Tkinter. I did not include that in this pull request because it would add a new dependency and mixes two GUI libraries, but I can include that if it's wanted. 

To clarify, you prefer ALL files being deleted to be listed? Not just the type from the path or the number of files?

**Comment by zas** [comment]
This PR contains unrelated commits

**Comment by samj1912** [comment]
This PR has been sitting untouched for more than 11 months now. Closing it with a 'Needs more work' tag.
You can reopen this if you want to resume working on it again. Be mindful we have updated to Qt5 and Py3 now.

**Pull request closed without merging**